ga4gh/cli.py

```diff
@@ -127,6 +143,32 @@ def printVariant(self, v):
         print()
 
 
+class ReferenceSetSearchRunner(AbstractSearchRunner):
+    """
+    Runner class for the referencesets/search method.
+    """
+    def __init__(self, args):
+        super(ReferenceSetSearchRunner, self).__init__(args)
+        request = protocol.GASearchReferenceSetsRequest()
+        if args.accessions is not None:
+            request.accessions = args.accessions.split(",")
+        if self.usingWorkaroundsFor('google'):
```

Comment:
I'm not crazy about using hard-coded values for program flow, as it's very brittle. Perhaps we could add a constant to `client.py` that we could check against here and in `client.py` instead?